doc/development.md

```diff
@@ -54,6 +53,23 @@ To upgrade your system with local build's RPM:
     make upgrade
 
 
+## Testing
+
+To run tests, first enter the virtual environment:
+
+    source ~/.venv/vdsm/bin/activate
+
+Then start some tests with tox, for example the networking tests:
+
+    tox -e network
+
+To exit the virtual environment afterwards:
+
+    deactivate
+
+For more information about testing see [../tests/README.md](../tests/README.md).
```

Comment:
Actually I think I tried that and it didn't work. Is that supposed to be some github-magic, that turns that into an absolute link? I'll try it again.

Reply:
It needs to start with the / in order to work. But that is better than relative paths.

Reply:
I just copied the URL to the page, no magic.

    [tests/README.md](https://github.com/oVirt/vdsm/blob/master/tests/README.md)

Reply:
I don't think that's a good idea, because then the link doesn't work inside a fork of the repo. It jumps back to the original. Do we want that?

Reply:
Well if you find a way to do a link that works both in a fork and in the main repo it would be better.

Reply:
I think `[/tests/README.md](/tests/README.md)` is fine, it works everywhere and it looks correct. Without the leading / it feels wrong, because we are in a subfolder that we need to get out of.